pkg/rpcclient/notary/actor.go

```diff
@@ -322,7 +323,8 @@ func (a *Actor) SendRequestExactly(mainTx *transaction.Transaction, fbTx *transa
 // see actor.Actor and actor.Waiter documentation for details. Wait may be used as a wrapper
 // for Notarize, SendRequest or SendRequestExactly.
 func (a *Actor) Wait(mainHash, fbHash util.Uint256, vub uint32, err error) (*state.AppExecResult, error) {
-	if err != nil {
+	// #2248 will eventually remove this garbage from the code.
+	if err != nil && !(strings.Contains(strings.ToLower(err.Error()), "already exists") || strings.Contains(strings.ToLower(err.Error()), "already on chain")) {
```

Comment:
It's a good change, but at the same time I'm thinking about returning some `neorps.ErrAlreadyExist` error right from the client, because in neofs-node we have https://github.com/nspcc-dev/neofs-node/commit/816c74d1857f2f0940af27da029413b3716c2eeb (and IIUC, this code  won't work now).

We also have almost the same code [here](https://github.com/nspcc-dev/neofs-node/blob/816c74d1857f2f0940af27da029413b3716c2eeb/pkg/morph/client/notary.go#L827), and this change may affect users when they will migrate on actor.

Or we probably need to document this behaviour somewhere.

Reply:
`ErrAlreadyExists` kinda works for Go RPC node, but won't work for C# one which may be a problem for generic waiter. Some documentation for this behavior can be helpful though.

Reply:
Updated doc.